### PR TITLE
feat: add features to dashboard

### PR DESCRIPTION
**Pull request by Shurtu-gal**

### CODE REVIEW

File: client/src/components/dashboard-navbar.tsx [hunk 1]
```diff
@@ -119,6 +120,23 @@ export const DashboardNavbar: React.FC<IDashboardNavbar> = ({ onSidebarOpen, ...
...
+              display: {
+                xs: 'none',
+                sm: 'inherit',
+              },
+            }}
+            size="small"
+            startIcon={<AddCircleOutlineOutlined />}
+            onClick={() => setOpenCreateOrgModal(true)}
+          >
+            Add Organisation
```
Comment (riteshsp2000) [inline client/src/components/dashboard-navbar.tsx:142]: Have you checked all the permissions and necessary things needed here? Organization is the top most item in the hierarchy and  as much i remember, only super admins could add a new organization.

File: client/src/components/product/product-create-modal.tsx [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+      return {
+        ...current,
+        [key]: current[key],
+      };
+    });
+  };
+
+  const handleEventCreate = async () => {
+    try {
+      await createEvent({
```
Comment (riteshsp2000) [inline client/src/components/product/product-create-modal.tsx:96]: Same here as well. Only this particular organization members with admin access should see the button and be able to add new events

**Review by riteshsp2000 — APPROVED** [review]

File: client/src/components/dashboard-navbar.tsx (hunk 1 above)
Comment (Shurtu-gal) [reply client/src/components/dashboard-navbar.tsx:142]: Sorry, forgot to add it at these places.

File: client/src/components/product/product-create-modal.tsx (hunk 2 above)
Comment (Shurtu-gal) [reply client/src/components/product/product-create-modal.tsx:96]: It has been managed here https://github.com/dscnitrourkela/project-dates-web/blob/933e7b3db5dd7a140bc2dcbcdbf83716c8865e9a/client/src/components/product/product-list-toolbar.tsx#L34.

Comment (Shurtu-gal) [reply client/src/components/product/product-create-modal.tsx:96]: It has been managed here https://github.com/dscnitrourkela/project-dates-web/blob/933e7b3db5dd7a140bc2dcbcdbf83716c8865e9a/client/src/components/product/product-list-toolbar.tsx#L34-L43

File: client/src/components/dashboard-navbar.tsx (hunk 1 above)
Comment (Shurtu-gal) [reply client/src/components/dashboard-navbar.tsx:142]: Done 
https://github.com/dscnitrourkela/project-dates-web/blob/10d7ff9711d5958ead08eb4aae584b58f59ed336/client/src/components/dashboard-navbar.tsx#L127-L143

**Pull request merged**